packages/blade/src/components/Card/CardSurface.native.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components/native';
+import { useTheme } from '~components/BladeProvider';
+import Box from '~components/Box';
+import type { BoxProps } from '~components/Box/types';
+
+const CardSurfaceStyled = styled(Box)<{ elevation: number; surfaceLevel: 2 | 3 }>(
+  ({ surfaceLevel, theme }) => {
+    const backgroundColor = theme.colors.surface.background[`level${surfaceLevel}`].lowContrast;
+    return {
+      shadowOpacity: '1',
```

Comment:
let's use this from token itself? its commented out there but we shall uncomment it there now

Reply:
There is no shadowOpacity token I believe. It's just 1. 🤔

Reply:
yes but we have that value in our tokens. [check this](https://github.com/razorpay/blade/blob/master/packages/blade/src/tokens/theme/paymentTheme.ts#L2202)

Reply:
Ahh i see. but It's commented should we expose it?